### PR TITLE
Feature/processing data

### PR DESCRIPTION
**Pull request by Buyaki01**
- Create model classes for all entities
- Set up associations between models
- Familiarize yourself with ['rails c' command](https://guides.rubyonrails.org/command_line.html#bin-rails-console)
- Open rails console for your project: Create at least one user, Create at least 4 posts written by one of the users and Create at least 6 posts comments for one of the posts you created
- Use other [CRUD methods](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/active_record_crud.md) to find, update and delete entities
- Add a method that returns the 3 most recent posts for a given user
- Add a method that updates the posts counter for a user and a method that returns the 5 most recent comments for a given post
- Add a method that updates the comments counter for a post
- Add a method that updates the likes counter for a post
- Make current user data available to all controllers by creating a method called current_user in your ApplicationController and the current_user method should return the first user from a database

### CODE REVIEW

File: app/models/application_record.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
+  def current_user
+    User.first
+  end
```
Comment (DeliceLydia) [inline app/models/application_record.rb]: - [ ] This method is supposed to be in the `ApplicationController` please address that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Buyaki01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Kudos to the good README
- Good job on setting up well associations
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/models/application_record.rb (hunk 1 above)
Comment (rloterh) [reply app/models/application_record.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ritta, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**